frontend/src/pages/PipelineDetails.tsx

```diff
@@ -325,8 +325,8 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
               <div className={css.containerCss}>
                 <Editor
                   value={templateString || ''}
-                  height='100%'
-                  width='100%'
+                  height='640px'
```

Comment:
Thank you for the fix!
But can you explain why it didn't work in safari, and why do we need to hard-code a number here?
There might be better solutions that do not need a hard-coding, but also cross browser.

Reply:
Thanks @Bobgy, for your quick response!
The reason of the fixes refer to the link here: https://github.com/securingsincity/react-ace/issues/235
I tried the solutions mentioned in the link, but unfortunately, it's not work, we may have to make a fixed value there to show the yaml file with the scroll bar.

Reply:
@Bobgy , is that ok to you?

Reply:
Thanks @Bobgy , for your solutions.
1. I tried to upgrade react-ace to different versions (7.0.4, 7.0.5, 8.1.0, 8.0.0, 9.1.0, 9.2.0 etc..), but all of them couldn't work, I don't think react-ace addressed this issue in Safari.

2. **Finally, I used the solution-2 you proposed.**  It works after my fully tested, please help check the code. Whether it's OK to you now? Thanks very much!